lib/board.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :grid
+
+  def initialize
+    @grid = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def display
+    puts '                       Welcome to Tic Tac Toe Game                  '
```

Comment:
#### `REQUIRED`

Let's move all user interaction (`puts`, `print`, `gets.chomp`, `p`, etc) to `main.rb`.

> Add **executable** file named `main.rb` to `bin` directory. **This file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` (`print` and `p`) can be used.**